### ISSUE TITLE
Milestone - Expanded functional search capabilities (4.11)

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/issues#522]
Enhancing functional search in the NMDC Data Portal
Search functionalities are one of the key value adds that NMDC brings to the community. In the Pilot, we
first developed KEGG for functional search capabilities despite the much larger set of functional
annotations that are produced from the metagenome workflow. Through our user research activities, we
will evaluate and prioritize additional functional search capabilities with our current annotation outputs
(Data Portal, Milestone 4.11). To make the functional annotation FAIRer, easier to use, and better suited
to multi-omics questions, we will add support for well-established ontologies (Milestone 2.12). We will
use the GO system77 as the ‘hub’ identifiers for function terms, and has compatibility advantages with
other systems and mappings across other annotation systems (e.g., KEGG, EC, Rhea, Panther, Pfam,
Interpro). It also has extensive synonyms to assist in user search, well-vetted hierarchies to allow for broad
search, and interlinks to the CHEBI chemical system78 such that concepts like ‘nitrate reduction’ and
‘nitrate’ are coupled.
Page 37

work to start in FY24 Q4, due FY25 Q1.

**Comment by ssarrafan** [comment]
From quarterly report:
This has been delayed due to other higher priorities across schema, workflow, and NEON data processing. Will plan to revisit expanded search features in connection with GO annotation plan (Milestone 2.12). Work to start Q4 of this year.

Changing to Q4.

**Comment by aclum** [comment]
@ljohnson09  What was the most requested after KEGG and GO base on the data portal user research? Pfam or COG?

**Comment by ljohnson09** [comment]
@aclum They were pretty even (6 for Pfam but with the qualifier that pfam was primarily used as the underlying DB for a tool or software for 2; then COG had 5 mentions)

**Comment by ssarrafan** [comment]
REscheduled to 2025 Q1 per the [milestones roadmap doc](https://docs.google.com/spreadsheets/d/1sc7iaAuU7LREs6evpPEvWNNcFYLzORM2Cj57036xHvE/edit?gid=948982184#gid=948982184)

**Comment by aclum** [comment]
This milestone will be complete as of the November 2024 production release, scheduled for November 19, 2024.
For the quarterly report: We made backend and front end changes to expand functional support. Based on user research we added support in the data portal for search for COG terms, functions, and pathways and for Pfam terms and clans.  We also expanded support for KEGG pathways, addressed bugs with linking to external references (KEGG pathways and Pfam clans) and removed references to KEGG BRITE hierarchies.
